src/features/serveStatic/ServeStatic.tsx

```diff
@@ -2,13 +2,15 @@ import http, { Server } from 'http'
 import serveStatic from 'serve-static'
 import path from 'path'
 
-import { glitch, squoosh } from '../constants/ServeStatic'
+import { glitch, squoosh, ffmpegwasm } from '../constants/ServeStatic'
 
 const servers: Server[] = []
 
 export default function initServeStatic(isPackaged: boolean): void {
   let squooshStaticPath = `${process.cwd()}/node_modules/squoosh/production`
   let glitchStaticPath = `${process.cwd()}/node_modules/jpg-glitch/production`
+  let ffmpegStaticPath = `${process.cwd()}/node_modules/ffmpegwasm-create-video/production`
```

Comment:
Could you explain why we need to service a library in such way?
Would be great if you place this info to comment near this code for future developers

Is it for code splitting? If so, can we use more convenient dynamic import instead?

Maybe some problem with webpack, or because of some electron specifics?

Reply:
I was faced with this issue `Electron can't load with ERR_WORKER_PATH` when I have tried to use `ffmpeg.wasm` and add it to the Electron app. Refer: https://github.com/ffmpegwasm/ffmpeg.wasm/issues/85 and I have a solution here https://github.com/mmamedel/ffmpegwasm-electron-test 

So that's why I have forked this repo https://github.com/ffmpegwasm/react-app from the owner and then custom it as a service to create the video.